CONTRIBUTION.md

```diff
@@ -63,6 +63,24 @@ We use [pytest](https://docs.pytest.org/en/latest/) for running tests. You can r
 poetry run pytest tests/
 ```
 
+There are two types of tests: those testing the internal functions and methods of the library and those testing the test execution with example queries. The latter require a running database instance to be available (either locally or remotely). Note, integration testing is currently only supported for Clickhouse. You can run internal testing only with:
+
+```bash
+poetry run pytest -m "not integration" tests/
+```
+
+Running integration tests locally requires [Docker](https://www.docker.com/) and [Docker Compose](https://docs.docker.com/compose/). First, start the local database services with:
+
+```bash
+docker compose up -d
```

Comment:
**question:** Since I always used `docker-compose.yml` files in stead of only `compose` - does docker-compose pick up that other file as well?